### ISSUE TITLE
Doesn't work if arg is nullable in PHP

### ISSUE THREAD

**Issue by alozytskyi** [xylo/intellij-postfix-templates#115]
Example: 

exp is a template(I was lazy to copy symbol so -> is replaced with symbol): 

```
.exp : Explode string
  string -> explode('$DELIMITER$', $expr$)
```

Input:

```php
function convertPath(?string $path = null) {
   return (new \DS\Vector($path.exp<TAB>))
}
```

Expected

```php
function convertPath(?string $path = null) {
   return (new \DS\Vector(explode('<CURSOR>', $path)))
}
```

Reality: 

Doesn't work, if I add: `$path = (string) $path`, then it works.

**Comment by xylo** [comment]
Fixed in version 2.10.1.

**Comment by xylo** [comment]
I hope these changes in type matching do not have any side effects.  If you discover type mismatch, please let me know.
